src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -888,6 +888,17 @@ public function register(string $id, ?string $class = null): Definition
         return $this->setDefinition($id, new Definition($class));
     }
 
+    /**
+     * Registers a service definition.
+     *
+     * This method allows for simple registration of service definition
+     * with a fluid interface.
+     */
+    public function registerChild(string $id, string $parent): Definition
```

Comment:
This change is unrelated, could you propose it in another PR to keep this one focused?

Reply:
Done, see #58004

Reply:
#58004 has been merged now, @HypeMC Can you rebase this one when you have time?

Reply:
@fabpot Done